src/neo/Network/P2P/Payloads/Transaction.cs

```diff
@@ -292,8 +292,6 @@ public virtual VerifyResult VerifyStateDependent(DataCache snapshot, Transaction
             foreach (UInt160 hash in hashes)
                 if (NativeContract.Policy.IsBlocked(snapshot, hash))
                     return VerifyResult.PolicyFail;
-            if (NativeContract.Policy.GetMaxBlockSystemFee(snapshot) < SystemFee)
```

Comment:
This is not moved to `ProtocolSettings`?

Reply:
Maybe it can be moved into the consensus settings?